json2html/jsonconv.py

```diff
@@ -5,6 +5,9 @@
 (c) Varun Malhotra 2013
 Source Code: https://github.com/softvar/json2html
 ------------
+Michel Müller 2015-1-31: Added bootstrap option, converting table-like JSON
```

Comment:
Please add this line `# coding=utf-8` at the beginning of this file. Was getting this error https://www.dropbox.com/s/n4vej605038wxte/Screenshot%202015-01-31%2020.31.46.png?dl=0